### DISCUSSION TITLE
OTA with ESPAsyncWebserver Server Instance

### DISCUSSION THREAD

**Discussion by JoJos1220** [JAndrassy/ArduinoOTA#214] [Q&A]
Hi all,

is it possible to rewrite the libary to use a given ESPAsyncWebserver server Instance to avoide server collision and for saving memory instead of adding seperatly libarys for server instance?

Regards, Jojo

**Comment by JAndrassy** [comment]
no, but you can rewrite the [WebUpdater](https://github.com/arduino/arduino-esp32/blob/master/libraries/HTTPUpdateServer/examples/WebUpdater/WebUpdater.ino) example for the ESPAsyncWebserver
